sections/changes.include

```diff
@@ -5,6 +5,29 @@
 
   Full details of all changes since 12 January 2016 are available from the <a href="https://github.com/w3c/html/commits/master">commit log</a> of the <a href="https://github.com/w3c/html/">w3c/html github repository</a>, including various editorial and linking fixes.
 
+  <h3 id="changes-wd5">Changes between <a href="https://www.w3.org/TR/html52/">Working Draft 6</a> and <a href="https://www.w3.org/TR/2017/WD-html52-20170228/">Working Draft 5</a></h3>
+  
+  <dl>
```

Comment:
Missing changes to menu/menuitem:

[Make menu and menuitem match reality](https://github.com/w3c/html/commit/0b959a586e54fe4145b69303ebb11755cea26c09)
Substantive changes:
*remove `HTMLElement.contextmenu` IDL attribute
*remove the `radio` and `checkbox` types for menus
*Make `menuitem` have content, as per implementation.
*A `menuitem` makes a command if it is labeled
*change terminology from "popup menu" state to "context menu" state.